active/39-group-chat/README.md

```diff
@@ -16,7 +16,7 @@ Provide a powerful group chat with flexible moderation features
 - Testing & Evaluation: (TBD)
 - Contributor: @themue
```

Comment:
It makes sense for you to swap with me as the lead contributor for the group chat swarm @themue, my interests lay more with chat moderation.